cmd/alerts/list_alerts.go

```diff
@@ -0,0 +1,124 @@
+package alerts
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+var levelCmd string
+
+type alertCmd struct {
+	clusterID  string
+	alertLevel string
+}
+
+type Labels struct {
+	Alertname string `json:"alertname"`
+	Severity  string `json:"severity"`
+}
+
+type Status struct {
+	State string `json:"state"`
+}
+
+type Annotations struct {
+	Summary string `json:"summary"`
+}
+
+type Alert struct {
+	Labels      Labels      `json:"labels"`
+	Status      Status      `json:"status"`
+	Annotations Annotations `json:"annotations"`
+}
+
+// osdctl alerts list ${CLUSTERID} --level [warning, critical, firing, pending, all] --active bool
+func NewCmdListAlerts() *cobra.Command {
+	alertCmd := &alertCmd{}
+	newCmd := &cobra.Command{
+		Use:               "list <cluster-id>",
+		Short:             "list alerts",
+		Long:              `Checks the alerts for the cluster`,
+		Args:              cobra.ExactArgs(1),
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			alertCmd.clusterID = args[0]
+			ListAlerts(alertCmd)
+		},
+	}
+
+	newCmd.Flags().StringVarP(&alertCmd.alertLevel, "level", "l", "", "Alert level [warning, critical, firing, pending, all]")
+	return newCmd
+}
+
+func alertLevel(level string) string {
+	switch level {
+	case "warning":
+		levelCmd = "warning"
+	case "critical":
+		levelCmd = "critical"
+	case "firing":
+		levelCmd = "firing"
+	case "pending":
+		levelCmd = "pending"
+	case "all":
+		levelCmd = "all"
+	default:
+		log.Fatalf("Invalid alert level: %s\n", level)
+		return ""
+	}
```

Comment:
This is completely personal, subjective preference, but if you wanted you could rewrite this switch as follows.
```suggestion
	switch level {
	case "warning", "critical", "firing", "pending", "all":
		levelCmd = level
	default:
		log.Fatalf("Invalid alert level: %s\n", level)
		return ""
	}
```